lib/logstash-filter-geoip_jars.rb

```diff
@@ -0,0 +1,8 @@
+# this is a generated file, to avoid over-writing it just delete this comment
```

Comment:
This file should be removed and not checked in as it is generated automatically when the vendor_jars command is run.